hipposerve/api/app.py

```diff
@@ -43,14 +44,14 @@ async def lifespan(app: FastAPI):
             )
 
         await user.set({users.User.api_key: SETTINGS.test_user_api_key})
-        print(
-            f"Created test user: {user.name} with API key: {user.api_key}. "
-            "You should NOT see this message in production."
+        logger.warning(
+            f"Created test user: {user.name} with API key: {user.api_key}, "
```

Comment:
the standard way of using a logger is to let the logger handle string interpolation, so:

    logger.warning("Created test user: {} with API key: {}", user.name, user.api_key)

however it is probably easier to just use `f-strings` directly, so I do not feel strongly about this.

Reply:
I've changed all the logging calls to not use f-strings, thanks!